third-party/gmp/Makefile

```diff
@@ -10,7 +10,7 @@ include $(CHPL_MAKE_HOME)/make/Makefile.base
 #
 GMP_CROSS_COMPILED=no
 ifneq (, $(filter cray-x%,$(CHPL_MAKE_TARGET_PLATFORM)))
-CHPL_GMP_CFG_OPTIONS += --host=x86_64-cle-linux-gnu
+CHPL_GMP_CFG_OPTIONS += --host=$(shell uname -m)-cle-linux-gnu
```

Comment:
It's a little unclear to me whether `uname -m` or `uname -p` is what we want here.  `git grep` in the gmp-src subdir seems to imply that `uname -p` might be what's needed, but I'm far from certain about that.

Reply:
You aren't the only one to wonder about it.  It varies depending on the type of system.  Solaris man pages say to use `uname -p` and not `uname -m`.  Linux man pages say `uname -p` is not portable.  On a Mac, `uname -p` yields i386 and `uname -m` yields x86_64.

It turns out that on the current Cray systems, `uname -p` and `uname -m` return identical results (x86_64 or aarch64).  Since Cray systems run Linux, I followed the Linux `uname` man page and avoided `uname -p` because they say it is nonportable.  However, no matter what I choose, there is no guarantee we won't have to revisit it someday.

Reply:
By the way, in all the third-party packages, they use both `uname -p` and `uname -m` in their configure scripts.

Reply:
Based on their saying that `uname -p` is nonportable I'd say stick with your original `uname -m`, then.  Good to go!